### ISSUE TITLE
Class cast error

### ISSUE THREAD

**Issue by prenagha** [jvmlet/reveal-dependency-plugin#2]
Getting class cast exception below when try to run in IntelliJ 2023.2.1 using Gradle 8.2.1, and selecting dependency view on a task in the Gradle Tool Window.

```
Cannot cast object 'org.gradle.execution.plan.DefaultFinalizedExecutionPlan@4e3c87ee' 
with class 'org.gradle.execution.plan.DefaultFinalizedExecutionPlan' 
to class 'org.gradle.execution.plan.DefaultExecutionPlan'
```

**Comment by jvmlet** [comment]
Hmmm, this idea plugin basically executes this Gradle plugin `com.dorongold.plugins:task-tree:2.1.0` to render dependency tree. Can you try to execute the latest 2.1.1 version ?
If still fails, please open the issue at his repo https://github.com/dorongold/gradle-task-tree

**Comment by dorongold** [comment]
@jvmlet as you suggested above, this is indeed already solved in version 2.1.1 of the `task-tree` plugin.
I created a pull request here that bumps `task-tree` version to 2.1.1.
